src/main/kotlin/io/easybreezy/project/api/Router.kt

```diff
@@ -135,6 +130,21 @@ class Router @Inject constructor(
         }
 
         route("/{slug}") {
+
+            authorize(setOf(Activity.ISSUES_SHOW), { memberHasAccess(setOf(Activity.ISSUES_SHOW)) }) {
+                get<Response.Listing<Issue>, SlugParam>("/issues") { params ->
+                    controller<IssueController>(this).list(params.slug)
+                }
+            }
+
+            authorize(setOf(Activity.ISSUES_MANAGE)) {
+                post<Response.Either<Response.Ok, Response.Errors>, NewIssue, SlugParam>("/issues/add") { new, params ->
```

Comment:
All members of the project can add issue. Maybe we should remove Activity.ISSUES_MANAGE

Reply:
I think there are should be only is project member checking

Reply:
ok